requirements.txt

```diff
@@ -13,6 +13,6 @@ pymongo==3.11.3
 python-dateutil==2.8.1
 s3transfer==0.3.6
 six==1.15.0
-typing-extensions==3.7.4.3
+typing-extensions==3.10.0.
```

Comment:
you've an extra dot at the end, or a missing number

Reply:
changed version to 3.10.0.0